client/lcd/swagger-ui/swagger.yaml

```diff
@@ -521,9 +556,629 @@ paths:
+      required: true
+      type: string
+    get:
+      summary: Query the current delegation status between a delegator and a validator
+      tags:
+      - ICS21
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+        404:
+          description: Not Found
+        500:
+          description: Internal Server Error
+  /stake/delegators/{delegatorAddr}/unbonding_delegations/{validatorAddr}:
+    parameters:
+    - in: path
+      name: delegatorAddr
+      description: Bech32 AccAddress of Delegator
+      required: true
+      type: string
+    - in: path
+      name: validatorAddr
+      description: Bech32 OperatorAddress of validator
+      required: true
+      type: string
+    get:
+      summary: Query all unbonding delegations between a delegator and a validator
+      tags:
+      - ICS21
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+        404:
+          description: Not Found
+        500:
+          description: Internal Server Error
+  /stake/validators:
+    get:
+      summary: Get all validator candidates
+      tags:
+      - ICS21
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+        404:
+          description: Not Found
+        500:
+          description: Internal Server Error
+  /stake/validators/{validatorAddr}:
+    parameters:
+    - in: path
+      name: validatorAddr
+      description: Bech32 OperatorAddress of validator
+      required: true
+      type: string
+    get:
+      summary: Query the information from a single validator
+      tags:
+      - ICS21
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+        404:
+          description: Not Found
+        500:
+          description: Internal Server Error
+  /stake/pool:
+    get:
+      summary: Get the current state of the staking pool
+      tags:
+      - ICS21
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+        500:
+          description: Internal Server Error
+  /stake/parameters:
+    get:
+      summary: Get the current staking parameter values
+      tags:
+      - ICS21
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+        500:
+          description: Internal Server Error
+  /slashing/signing_info/{validator}:
+    get:
+      summary: Get sign info of given validator
+      description: Get sign info of given validator
+      produces:
+      - application/json
+      tags:
+      - ICS23
+      parameters:
+      - type: string
+        description: Bech32 validator public key
+        name: validator
+        required: true
+        in: path
+      responses:
+        '200':
+          description: OK
+        500:
+          description: Internal Server Error
+  /slashing/unjail:
```

Comment:
I think this path should be /slashing/validators/{validatorAddr}/unjail to be consistent with the restful approach of this API.

Reply:
Yes, good catch.